### ISSUE TITLE
decimal records are not processed correctly

### ISSUE THREAD

**Issue by crepon-philipp** [martijn/xsv#16]
I have a file containing multiple decimal values.
`0.001 | 0.01 | 0.1`

when parsing the data as described in README the first decimal is not returned correctly!

This is what I do:
```
x = Xsv::Workbook.open("sheet.xlsx")
sheet = x.sheets[0]
sheet.each_row do |row|
  p '---row', row
end
```

This is what I see in the console (the first value is just 1 instead of 0.001):
`[1, 0.01, 0.1]`

**Comment by martijn** [comment]
Could you share your file? My email address is in my profile.

**Comment by martijn** [comment]
Turns out the value was stored as a complex number in the XML (1E-3). Haven't seen that before and didn't notice it in the specification either. This is fixed in Xsv 0.3.16.
